# BIOL 342

## Review

monocistronic = only 1 gene in the mRNA

polycistronic: each gene has its own ribosome-binding site (SD + AUG)

polycistronic mRNA is not an operon because there is no control (no operator present on the gene) they are constitutive, aka housekeeping, genes

- genes become blurry... in an operon, only one promoter and one terminator controls production of *several proteins*... however, they are typically still considered distinct genes rather than one gene coding for several proteins

- shine-dalgarno isn't right at the 5' end, there's some space
- then some more space between SD and start codon (usually ~6–10 bp)
- in most cases, the fM (translated from the start codon) is cleaved off in post-translational processing

### POST-TRANSCRIPTIONAL MODIFICATION

- 5' guanine cap
- 3' polyadenine tail: contributes to stability

### POST-TRANSLATIONAL MODIFICATION

- often, Met/fMet is removed

### STRUCTURAL DIFFERENCE BETWEEN DNA AND mRNA

- mRNA does not have Promoter or Terminator (those belong to DNA only)—well, part of the Terminator remains, but not the entire thing
- start codon in mRNA is AUG, not ATG (also ATG has no impact on DNA)
- shine-dalgarno sequence exists in both, but only has an impact on translation (mRNA → protein)
- codons are only ever in the CDS / ORF
  - eg. cannot call the nucleotides in the SD a codon, because that region is not translated

## Set 2

### RFLP Analysis

- for finding genetic differences between *closely related strains*
- useful in epidemiology: tracing movement of pathogen strains through a population
  - eg. you can see if a particular disease arose from one origin, or multiple foci

#### how it works

- tests *DNA*
- digest entire genome with a specific suite of restriction enzymes
- look for differences in the resulting pattern → different bands mean that there's a differing number of bps between the same restriction sites → the sequence between those two organisms in that region is different

#### what does the presence of a different-looking band mean?

possibilities:

- there was a deletion or SNP in the restriction site, therefore removing it (the enzymes no longer recognize)
- transposons: less bands, that are smaller
- insertion that causes a new restriction site: more, smaller bands

![rflp analysis](/resources/uw/images/rflp.jpg){:width="100%"}

in image above:

- B and C are likely different from D, E F
- D and F are likely different from E
- E is more similar to F than D

** careful not to use the phrase "the same" in describing the bands, because they are likely not the same (RFLP cannot tell us that level of detail)

### Cloning Vectors

#### size

naturally occuring plasmids range from 1 kb to >500 kb! (however, the ones used for cloning are usually small (<12 kb))

larger plasmids can hold more genetic information, but are also more metabolically demanding

#### copy number

multiple copies of a plasmid will exist in the host, usually from <10 to >100 copies

some low-copy number plasmids contain information to ensure they will be distributed to all new daughter cells (segregation). this ensures they are perpetuated, otherwise, they would quickly be lost within a population.

#### expression vs cloning vectors

### Cloning Vectors

#### features of all modern cloning vectors

- origin of replication (eg. OriC in e. coli)
- selectable marker (ampicillin resistance gene, etc.) OR screening marker (different from selectable)
- multiple cloning site (MCS)
- small size
- high copy number (this goes hand in hand with small size. if the size is big the cell will have a hard time getting the energy to maintain so many copies)

#### example: 
